nuxeo-core/nuxeo-core-schema/src/main/java/org/nuxeo/ecm/core/schema/FacetNames.java

```diff
@@ -105,6 +105,14 @@
      */
     public static final String HAS_RELATED_TEXT = "HasRelatedText";
 
+    /**
+     * The document content will be stored as a cold (cool) storage, this is the case where the content represents:
```

Comment:
The document content will be stored **in a** cold (cool) storage, this is the case **when** the content represents:

Reply:
my idea here is to explain that this facet had the purpose of indicating that the content is stored as archive ... -> `he document content will be stored as a cold (cool) storage` 
`this is the case where the content` here i talk about the case ... where this facet had a purpose